backend/backend/internal/mocks/github.com/prometheus/client_golang/api/prometheus/v1/mock_API.go

```diff
@@ -0,0 +1,1376 @@
+// Code generated by mockery. DO NOT EDIT.
```

Comment:
Hm, why is this in here?

Reply:
hm no idea? i ran `mockery` from the cli when i was trying to create a mock for testing but not sure why that got picked up? Shall i delete it?

Reply:
Hm, maybe try deleting this.
I just ran mockery from the tip of main and saw no changes, so it's unclear to me why this is here.